boot/bootutil/include/bootutil/bootutil.h

```diff
@@ -57,6 +60,17 @@ struct boot_rsp {
     uint32_t br_image_off;
 };
 
+/* This is not actually used by mcuboot's code but can be used by apps
```

Comment:
Well, I agree that API could use this but I'll wait for this one to be merged and send another PR over that one or update the existing PR if it's not merged by then (most probable situation!).

Btw, I added the original `struct` as proposed and added `offsetof` asserts to guarantee the relevante offsets are not changed in the code. That `struct` is supposed to be used by "clients". Creating an include with only "exported" stuff seems like a good idea.